project/plugins.sbt

```diff
@@ -1,4 +1,5 @@
-addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-RC12")
+addSbtPlugin("ch.epfl.scala" % "sbt-release-early" % "2.0.0")
```

Comment:
This is good, but I'd depend on the latest `2.0.0+7-afca4e95`, because it's stable.

Reply:
But we'll need to read the git log to learn what's new in this latest version 😜